cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/AnalyzerUtils.java

```diff
@@ -182,6 +183,19 @@ public static int compare(double d1, double d2, double epsilon) {
     return caseInsensitiveGoalsByName;
   }
 
+  /**
+   * Get a replica movement strategy map with strategy name as the keys.
+   */
+  public static Map<String, ReplicaMovementStrategy> getCaseInsensitiveReplicaMovementStrategiesByName(KafkaCruiseControlConfig config) {
```

Comment:
we already have a Scala class named `ExecutorUtils`,I merged this function with `KafkaCruiseControl#getReplicaMovementStrategy` since I feel we will not reuse this logic in other places.